.vib/spring-cloud-dataflow/cypress/cypress/support/spring_cloud_dataflow_spec.js

```diff
@@ -1,6 +1,9 @@
 /// <reference types="cypress" />
```

Comment:
As said, I reckon this file belongs to the `integration` folder.

Reply:
Oops, great catch. I fixed this in both instances.